tests/test-property-propagation.stanza

```diff
@@ -0,0 +1,112 @@
+#use-added-syntax(jitx, tests)
+defpackage ocdb/test/property-propagation : 
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/property-propagation 
+
+; Tests the 
+deftest test-unique-nets : 
+  val num-instances = 3
+
+  pcb-component Component : 
+    port p: pin[[1 2]]
+
+  pcb-module Module : 
+    public inst c : Component[num-instances]
+
+  pcb-module main-module : 
+    inst m1 : Module
+    inst m2 : Module
+
+    net GND ()
+    net (m2.c[1].p[2], m1.c[1].p[2])
+    for n in 0 to num-instances do : 
+      net (m1.c[n].p[1], m2.c[n].p[1])
+      net (m1.c[n].p[2], m2.c[n].p[2])
+      net (m1.c[n].p[2], GND)
+    
```

Comment:
This also seems in conflict with line 24 and line 27? Or is your intent to connect all p[2] pins to GND?